.github/workflows/sanity-checks.yml

```diff
@@ -0,0 +1,86 @@
+on:
+  # push:
+  #   branches: [main, 'release-v**']
```

Comment:
Reminder: Needs to be re-enabled.

Reply:
But why do we wan to run the sanity checks (AGAIN) in main? Or in a release branch?
Isn't it supposed to be code that has already pass all of these tests and proceed to building the wasm, the docker and the benchmarks (if applicable) but not run the entire checks suite again?

Reply:
TBH, that's a question I have asked myself. IMO, we can limit sanity checks to PRs against `main` in the future but not commits. WDYT @mustermeiszer @lemunozm ?

Reply:
I understand that if a PR is merged and jobs passed succesfully, then we do not need to run them again, as @gpmayorga says.

> we can limit sanity checks to PRs against main in the future but not commits.

What do you mean by "but not commits"?